apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -41,9 +41,19 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
 
     const searchAddress = useCallback(
         (query, skip) => {
+            const splitted_query = query.split(query_split_regex)
```

Comment:
Because stripping last space will solve only one extra space at the end. This solution solves the problem which is doesn't got our tech support, but will have in future. For example, you are searching "Ленина 48" and searching will be unsuccessfull, because this address in database is "Ленина д.48", so the string exact search will falls